editor/src/messages/tool/tool_messages/artboard_tool.rs

```diff
@@ -226,20 +226,7 @@ impl Fsm for ArtboardToolFsmState {
 
 							ArtboardToolFsmState::Dragging
 						} else {
-							let id = generate_uuid();
-							tool_data.selected_board = Some(id);
-
-							tool_data.snap_manager.start_snap(document, document.bounding_boxes(None, Some(id), font_cache), true, true);
-							tool_data.snap_manager.add_all_document_handles(document, &[], &[], &[]);
-
-							responses.push_back(
-								ArtboardMessage::AddArtboard {
-									id: Some(id),
-									position: (0., 0.),
-									size: (0., 0.),
-								}
-								.into(),
-							);
+							tool_data.selected_board = None;
```

Comment:
For clarity, please use the full name "artboard" to hedge against anyone in the future mistaking the meaning for something else.

Reply:
This was introduced before your MR @Kit-p however it would still be great if you could quickly rename this for clarity.

Reply:
Oh I overlooked that fact, thanks @0HyperCube for pointing out that @Kit-p didn't choose that name to begin with. Yes, please do rename it for clarity while you're at it, thank you!

Reply:
No problem, renamed.